openfisca_survey_manager/scenarios.py

```diff
@@ -256,7 +256,7 @@ def compute_aggregate(self, variable = None, aggfunc = 'sum', filter_by = None,
                         f"{alternative_weights} is not a valid variable of the tax benefit system"
                     weight_variable = alternative_weights
 
-                elif type(alternative_weights) == int or type(alternative_weights) == float:
+                elif isinstance(alternative_weights, int) or isinstance(alternative_weights, float):
```

Comment:
This fix E721 lint error, I don't now how it could exist on master. Flake8 rules update ?